### PR TITLE
WiX: Drop the feature flag, always build with WiX support.

### PR DESCRIPTION
**Pull request by mook-as**
Note that we're still building the NSIS-based installer, so that older versions have something to upgrade to.  After the next release, we should be able to drop that (since the release would be looking for msi installers instead of exe installers at that point).

### CODE REVIEW

**Comment by ericpromislow** [comment]
Do we not support people skipping versions during an upgrade? Like going from 1.6.x (NSIS-based installer) to, say, 1.12.0

**Comment by mook-as** [comment]
They can, but it wouldn't be using the auto-update; they can still download the MSI installer manually at any point.

File: scripts/lib/sign-win32.ts [hunk 1]
```diff
@@ -30,9 +30,11 @@ const REQUIRED_WINDOWS_CONFIG = {
 const DEFAULT_WINDOWS_CONFIG = {
   certificateSha1:        '', // set via CSC_FINGERPRINT
   rfc3161TimeStampServer: 'http://timestamp.digicert.com',
+  extraFiles:             [] as Array<string>,
```
Comment (ericpromislow) [inline scripts/lib/sign-win32.ts:33]: Where is this field used? I pulled it out and could run `npm run sign -- "dist/Rancher Desktop.some-version.zip` without a problem.

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Just the one question on the `extraFiles` field. If it is needed, please add a comment explaining how, because it's not obvious following the function calls that use the `config` variable.

File: scripts/lib/sign-win32.ts (hunk 1 above)
Comment (mook-as) [reply scripts/lib/sign-win32.ts]: I had some other code to try to munge it, but it didn't workout and I missed removing this change.

Amended the commit to get rid of it.

**Review by ericpromislow — APPROVED** [review]
LGTM

**Pull request merged**